src/plugins/vis_type_timeseries/public/application/components/vis_types/markdown/vis.js

```diff
@@ -65,16 +54,18 @@ function MarkdownVisualization(props) {
     markdown = (
       <div className="tvbMarkdown" data-test-subj="tsvbMarkdown">
         {markdownError && <ErrorComponent error={markdownError} />}
-        <style type="text/css">{markdownCss}</style>
         <div className={contentClasses}>
-          <div id={markdownElementId}>
-            {!markdownError && (
-              <Markdown
-                markdown={markdownSource}
-                openLinksInNewTab={model.markdown_openLinksInNewTab}
-              />
-            )}
-          </div>
+          {Style.it(
```

Comment:
`style-it` is on old, unmaintained library which has security problems: https://github.com/elastic/kibana/issues/100639

It should be possible to do the same thing using `@emotion/react` - for example like this: https://github.com/elastic/kibana/blob/eef094bafb4ce265126495754f8cb1cf760bb614/src/plugins/chart_expressions/expression_tagcloud/public/expression_renderers/tagcloud_renderer.tsx#L60

Reply:
Hm, I'll try to use this

Reply:
Now I used `@emotion/react`